samples/ClassroomLabs/Modules/Library/Az.LabServices.psm1

```diff
@@ -1576,7 +1822,10 @@ function New-AzLabSchedule {
         [Array] $WeekDays = @('Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'),
 
         [parameter(Mandatory = $false, ValueFromPipelineByPropertyName = $true, HelpMessage = "Notes for the class meeting.")]
-        $Notes = ""
+        $Notes = "",
+
+        [parameter(Mandatory = $false, ValueFromPipelineByPropertyName = $true, HelpMessage = "Event type.")]
+        [System.String]$EventType = "Standard"

```

Comment:
Should we add a validate set attribute here?

Reply:
Yep!  Good idea, I'll add this